docs/src/usage.md

```diff
@@ -1,12 +1,93 @@
-# Usage
+# Using Kani
 
 At present, Kani can used in two ways:
- * [On a single file](./kani-single-file.md) with the `kani` command.
- * [On a package](./cargo-kani.md) with the `cargo-kani` command.
 
-Running [Kani on a single file](./kani-single-file.md) is quite useful for small
-examples or projects that don't use `cargo`.
+ * [On a single crate](#usage-on-a-single-crate) with the `kani` command.
+ * [On a Cargo package](#usage-on-a-package) with the `cargo kani` command.
 
-However, if you plan to integrate Kani in your projects, the recommended
-approach is to use [Kani on a package](./cargo-kani.md) because of its ability
-to handle external dependencies.
+If you plan to integrate Kani in your projects, the recommended approach is to use `cargo kani`.
```

Comment:
Can we add this sentence back:

This is due to its ability to handle external dependencies and the option add configurations via the `Cargo.toml` file.